dialogue-blocking-channels/src/main/java/com/palantir/dialogue/blocking/BlockingChannelAdapter.java

```diff
@@ -80,7 +80,12 @@ private BlockingChannelAdapter() {}
             BlockingChannelAdapterTask runnable =
                     new BlockingChannelAdapterTask(delegate, endpoint, request, settableFuture);
             try {
-                Future<?> future = executor.submit(runnable);
+                Future<?> future;
```

Comment:
I know there's the backoff scheduler, but I think that can be shared, it's just 1 thread?